### DISCUSSION TITLE
Arch crashes when choosing hyprland

### DISCUSSION THREAD

**Discussion by rus100lan** [hyprwm/Hyprland#5449] [Q&A]
не могу установить hyprland, установил я в arch linux все через archinstall, выбираю профиль ввожу пароль,
и у меня вылетает на выбор профиля, пробовал fedora linux там ошибку выдает. про nix os молчу...  помогите! 

I can’t install hyprland, I installed everything in arch Linux via archinstall, select a profile, enter the password, and it crashes when choosing a profile, I tried fedora linux and it gives an error. I’m silent about nix os... help!

**Comment by mightydoomguy** [comment]
> не могу установить hyprland, установил я в arch linux все через archinstall, выбираю профиль ввожу пароль, и у меня вылетает на выбор профиля, пробовал fedora linux там ошибку выдает. про nix os молчу... помогите!
> 
> I can’t install hyprland, I installed everything in arch Linux via archinstall, select a profile, enter the password, and it crashes when choosing a profile, I tried fedora linux and it gives an error. I’m silent about nix os... help!


Тоже с этим столкнулся когда ставил на виртуалку помог выбор другого окружения  я поставил крыску чрез archinstall и уже зайдя нормально чрез крысу поставил hyprland
